### PR TITLE
Feature/lvjav442 41

### PR DESCRIPTION
**Pull request by NazarHladysh**

### CODE REVIEW

File: src/main/java/com/softserve/ita/java442/cityDonut/repository/DonateRepository.java [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface DonateRepository extends JpaRepository<Donate, Long> {
+    List<Donate> getByProjectId(long id);
+//    @Query(value = "SELECT new com.softserve.ita.java442.cityDonut.model.DonatedUserProject(v.id, count(v.sum), sum(v.sum), v.project.id, v.user.id) FROM Donate v group by v.project.id, v.user.id",
+//            nativeQuery = false)
+//    List<DonatedUserProject> findDonatedUserProject();
```
Comment (nastiamazurak) [inline src/main/java/com/softserve/ita/java442/cityDonut/repository/DonateRepository.java:16]: delete comments, please

**Review by nastiamazurak — APPROVED** [review]

File: src/main/java/com/softserve/ita/java442/cityDonut/controller/DonateController.java [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 @RequestMapping(path = "/api/v1/donate")
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/controller/DonateController.java:16]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/mapper/project/ProjectByUserDonateMapper.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                .donateSum(donatedUserProject.getSum())
+                .build();
+    }
+
+    @Override
+    public ProjectByUserDonateDto convertToDto(Project model) {
+        return null;
+    }
+
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/mapper/project/ProjectByUserDonateMapper.java:28]: delete row

File: src/main/java/com/softserve/ita/java442/cityDonut/service/impl/DonateServiceImpl.java [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
+import com.softserve.ita.java442.cityDonut.model.Project;
 import com.softserve.ita.java442.cityDonut.repository.DonateRepository;
 import com.softserve.ita.java442.cityDonut.service.DonateService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.LinkedList;
+import java.util.List;
+
+
```
Comment (NazarRyvak) [inline src/main/java/com/softserve/ita/java442/cityDonut/service/impl/DonateServiceImpl.java:15]: delete row

**Review by NazarRyvak — APPROVED** [review]

**Pull request merged**